### ISSUE TITLE
Add quotes module

### ISSUE THREAD

**Issue by ghermans** [idevelopment/suitecrm_laravel#79]
General fields for this module

- [x] Create list view
- quote_number
- quote_title
- quote_stage
- grand_total
- valid_untill

**Comment by Tjoosten** [comment]
database fields?

**Comment by ghermans** [comment]
@Tjoosten , i am working on the views for this controller, however this is not a priority because we are still waiting on your guzzle library for #5 ..

**Comment by ghermans** [comment]
Also links can be found on the sugarcrm documentation links i gave you earlier
